packages/backend/server/migrations/20241129062332_workspace_invite_status/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateEnum
+CREATE TYPE "WorkspaceMemberStatus" AS ENUM ('Declined', 'Pending', 'NeedMoreSeat', 'UnderReview', 'Accepted');
+
+-- AlterTable
+ALTER TABLE "workspace_features" ADD COLUMN     "configs" JSON NOT NULL DEFAULT '{}';
```

Comment:
why? there is already a configs column in features table

Reply:
allows store feature-specific configs, such as whether to globally disable ai in team workspace

Reply:
over concerning